child_compassion/model/child_compassion.py

```diff
@@ -116,27 +135,72 @@ def _get_child_from_project(project_obj, cr, uid, ids, context=None):
              ('M', _('Male'))], _('Gender')),
         'completion_date': fields.date(_("Completion date"),
                                        track_visibility="onchange"),
-        'desc_en': fields.text(_('English description')),
-        'desc_fr': fields.text(_('French description')),
-        'desc_de': fields.text(_('German description')),
-        'desc_it': fields.text(_('Italian description')),
-        'start_date': fields.date(_("Start date")),
+        # Dummy field which is overwritten in module sponsorship_compassion
+        'unsponsored_since': fields.date('Unsponsored since'),
+        # TODO : We store the descriptions in child database since we
+        # imported the descriptions from GP in this field. When all children
+        # will have a new case study fetched from Cornerstone,
+        # we can remove the field from db by removing store=True.
+        'desc_en': fields.related(
+            'case_study_ids', 'desc_en', type='text',
+            string=_('English description'), store={
+                'compassion.child.property': (
+                    _get_child_from_case_study,
+                    ['desc_en', 'desc_fr', 'desc_de', 'desc_it'],
+                    10)}),
+        'desc_fr': fields.related(
+            'case_study_ids', 'desc_fr', type='text',
+            string=_('French description'), store={
+                'compassion.child.property': (
+                    _get_child_from_case_study,
+                    ['desc_en', 'desc_fr', 'desc_de', 'desc_it'],
+                    10)}),
+        'desc_de': fields.related(
+            'case_study_ids', 'desc_de', type='text',
+            string=_('German description'), store={
+                'compassion.child.property': (
+                    _get_child_from_case_study,
+                    ['desc_en', 'desc_fr', 'desc_de', 'desc_it'],
+                    10)}),
+        'desc_it': fields.related(
+            'case_study_ids', 'desc_it', type='text',
+            string=_('Italian description'), store={
+                'compassion.child.property': (
+                    _get_child_from_case_study,
+                    ['desc_en', 'desc_fr', 'desc_de', 'desc_it'],
+                    10)}),
+        'has_desc_fr': fields.function(
+            _has_desc, string='FR', type='boolean', multi=True),

```

Comment:
The multi parameter is a group name. It seems to work with "True", but please chose a string to avoid side effects.